Tools/TestWebKitAPI/Tests/WebKitCocoa/SiteIsolation.mm

```diff
@@ -3001,4 +3001,22 @@ HTTPServer server({
         Util::spinRunLoop();
 }
 
+TEST(SiteIsolation, MainFrameRedirectBetweenExistingProcesses)
```

Comment:
Can we verify the web processes are correctly reused by checking `_webProcessIdentifier`?